.github/workflows/tests.yaml

```diff
@@ -0,0 +1,55 @@
+name: tests
+
+on: [pull_request, push]
+
+jobs:
+  lint:
+    name: Lint
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Install node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
+
```

Comment:
this job doesn't *need* to use the ljharb/node/install action, but it probably should for consistency. setup-node can't handle as many node versions properly.